packages/pixeloven/cli/src/commands/pixeloven.ts

```diff
@@ -2,12 +2,9 @@ import { PixelOvenToolbox } from "../types";
 
 export default {
     name: "pixeloven",
-    run: async (context: PixelOvenToolbox) => {
-        const { print } = context;
-        /**
-         * @todo Print out usage
-         * @todo we should be able to use the aliases and chain them.
-         */
-        print.info("Coming soon");
+    run: async (toolbox: PixelOvenToolbox) => {
+        const { print } = toolbox;
+        print.error(`Invalid argument`);
```

Comment:
Should prettier have changed this from a template string to a standard string?

Reply:
I guess it's not quite that smart cause I see others that don't have logic or variables as well. Carry on.